spec/single-spa.spec.js

```diff
@@ -1,750 +1,3 @@
-import * as singleSpa from '../src/single-spa.js';
-import * as SingleSpaChildAppError from '../src/single-spa-child-app-error.js';
+import './root-apps/systemjs-all-apps.spec.js';
 
-let Loader;
-
-function easyDeclareChild(name) {
```

Comment:
All of the deleted stuff got moved into the `.spec` files inside of the `spec/child-apps/` directory